### DISCUSSION TITLE
Is it possible to rename and/or move files included in an archive?

### DISCUSSION THREAD

**Discussion by hoisn** [goreleaser/goreleaser#1934] [Q&A] [answered]
E.g. I want to rename a release configuration file to a standard name or I want to move the user documentation folder just to be the released "doc" folder.

I didn't see something in the documentation, but goreleaser seems to be sometimes more powerful than that

Cheers
  Jan

**Comment by caarlos0** [comment, marked as answer]
Not possible at this moment, check https://github.com/goreleaser/goreleaser/issues/1638
